bbnclient/bbnclient.go

```diff
@@ -200,7 +200,7 @@ func (bbnClient *BabylonClient) isDelegationActive(
 		return false, nil
 	}
 
-	if uint32(len(btcDel.CovenantSigs)) < covQuorum {
+	if len(btcDel.CovenantSigs) < int(covQuorum) {
```

Comment:
just curious, why converting from uint to int (i.e. `covQuorum`) won't raise any gosec alarm?

Reply:
My fix here is actually following the code style below (206-210). My guess for gosec not showing alarm here is that covQuorum is a `uint32` var being casting to `int` and the cast does not specify it is a `int64` or `int32`. For `int` type, it really depends on whether you run on a 64-bit or 32-bit machine, so probably that is the reason the overflow conversion notice does show up.